src/world/Explosion.php

```diff
@@ -269,6 +271,51 @@ public function explodeB() : bool{
 		return true;
 	}
 
+	/**
+	 * Returns the explosion exposure of an entity, used to calculate explosion impact.
+	 */
+	public function getExposure(World $world, Vector3 $origin, Entity $entity) : float{
+		$bb = $entity->getBoundingBox();
+
+		$diff = new Vector3($bb->getXLength(), $bb->getYLength(), $bb->getZLength())->multiply(2)->add(1, 1, 1);
```

Comment:
```suggestion
		$diff = (new Vector3($bb->getXLength(), $bb->getYLength(), $bb->getZLength()))->multiply(2)->add(1, 1, 1);
```
We still support 8.1 on this branch

Reply:
Ah alright, resolved